exe-unit/src/handlers/local.rs

```diff
@@ -117,6 +117,37 @@ where
     }
 }
 
+impl<R: Runtime> Handler<SignExeScript> for ExeUnit<R> {
+    type Result = <SignExeScript as Message>::Result;
+
+    #[cfg(feature = "sgx")]
+    fn handle(&mut self, msg: SignExeScript, _: &mut Context<Self>) -> Self::Result {
+        let batch = self.state.batches.get(&msg.batch_id).ok_or_else(|| {
+            Error::RuntimeError(format!(
+                "signing an unknown ExeScript (batch [{}])",
+                msg.batch_id
+            ))
+        })?;
+
+        let script = batch.exe_script.clone();
+        let json = serde_json::to_string(&script)?;
```

Comment:
Is the serialization deterministic?

Reply:
Yes, it will match the output generated by this command